.github/workflows/on_pull.yml

```diff
@@ -0,0 +1,30 @@
+name: Node CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.14.1]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: npm install, build, and test
+      run: |
+        cd frontend
+        npm ci
+        npm run lint
+        npm test
```

Comment:
Nice it works - can we add `npm run typecheck` as another check? I think it would make the most sense after the linting check.